recipes/LibriSpeech/ASR/transducer/hparams/conformer_transducer.yaml

```diff
@@ -63,14 +63,33 @@ precision: fp32 # bf16, fp16 or fp32
 batch_size: 8
 grad_accumulation_factor: 4
 sorting: random
-avg_checkpoints: 1 # Number of checkpoints to average for evaluation
+avg_checkpoints: 10 # Number of checkpoints to average for evaluation
 
 # Feature parameters
 sample_rate: 16000
 n_fft: 512
 n_mels: 80
 win_length: 32
 
+# Streaming & dynamic chunk training options
```

Comment:
Any "DCT" acronym should be replaced by DynChunkTrain or something similar. The reason is that DCT means "Discrete Cosine Transform".